internal/security/setup/option/optionHelpers.go

```diff
@@ -214,20 +233,64 @@ func markComplete(dirPath string) error {
 	return nil
 }
 
-func getXdgRuntimeDir() string {
+func getWorkDir() string {
+	var workDir string
+	var err error
+
 	if xdgRuntimeDir, ok := os.LookupEnv(envXdgRuntimeDir); ok {
-		return xdgRuntimeDir
+		workDir = filepath.Join(xdgRuntimeDir, pkiInitBaseDir)
+	} else if setup.Configuration.SecretsSetup.WorkDir != "" {
+		workDir, err = filepath.Abs(setup.Configuration.SecretsSetup.WorkDir)
+		if err != nil {
+			// use default work dir if error occurs
+			workDir = filepath.Join(defaultWorkDir, pkiInitBaseDir)
```

Comment:
I would rather have this function return an error if there's an issue with the configuration.toml. 
Or at the very least log a warning because I have been bitten too many times by trying to understand why something isn't working when it's just silently ignoring my invalid configuration and using the default.

Reply:
yea, let's just return an error if operator makes  a mistake